gstreamer-encoder.c

```diff
@@ -401,7 +401,7 @@ bool gstreamer_encoder_encode(void *p, struct encoder_frame *frame,
 			}
 		}
 
-		data->codec_data = g_memdup(data->info.data, size);
+		data->codec_data = g_memdup2(data->info.data, size);
```

Comment:
Do we know which glib versions do have this symbol? E.g. what distributions are out there where this will break?

Reply:
g_memdup() has been deprecated since glib 2.68 (2021-03-18)...  How about if we were just to open code it instead with "g_malloc(); memcpy();" instead, given that there's only this one instance?

Reply:
malloc and copy sounds fine to me.

Reply:
Done.